docs/gds-supported-platform/getting-started-gsp-cluster.md

```diff
@@ -1,167 +0,0 @@
-# Getting started with a GDS Supported Platform cluster
```

Comment:
@samcrang does anything replace this deleted content?

Reply:
Not yet. We only want to remove it because it's confusing people, we definitely want to write something more up-to-date at some point.